src/appMain/smartDeviceLink.ini

```diff
@@ -202,6 +202,16 @@ TCPAdapterPort = 12345
 ; If the name is omitted, Core will listen on all network interfaces by binding to INADDR_ANY.
 TCPAdapterNetworkInterface =
 
+; WebSocket connection address and port
```

Comment:
update description

Reply:
Fixed in 32836e4d0626a660bbc5e7ae2a976dac1acb79ad